File: vb/decision/even-odd.vb
Module EvenOdd
    Sub Main()
        Dim n as Integer
        Console.Write("Enter a number: ")
        n = Console.ReadLine()
        If n mod 2 = 0 Then
            Console.WriteLine("Even Number")
        Else
            Console.WriteLine("Odd Number")
        End If
    End Sub
End Module

File: cs/decision/even-odd.cs
using System;

class EvenOdd
{
    static void Main()
    {
        int n;
        Console.Write("Enter a number: ");
        n = Convert.ToInt32(Console.ReadLine());
        if(n % 2 == 0)
        {
            Console.WriteLine("Even number");
        }
        else
        {
            Console.WriteLine("Odd number");
        }
    }
}